pkg/pillar/cmd/client/client.go

```diff
@@ -331,6 +351,11 @@ func Run(ps *pubsub.PubSub) { //nolint:gocyclo
 				done = true
 			}
 
+		case <-clientCtx.getCertsTimer.C:
+			// triggered by cert miss error in doGetUUID, so the TLS is device TLSConfig
+			ok := fetchCertChain(zedcloudCtx, devtlsConfig, retryCount, true)
+			log.Infof("client timer get cert chain %v\n", ok)
```

Comment:
Shouldn't you set gotServerCerts here if ok is set?

Reply:
we can, although if trigger is from doGetUUID(), we won't reach to that if gotServerCerts is not set